src/stores/AppStore.ts

```diff
@@ -66,15 +66,15 @@ class AppStore extends EventEmitter {
     }
 
     addCourse(newCourse: ScheduleCourse, scheduleIndex: number = this.schedule.getCurrentScheduleIndex()) {
-        let color: string;
+        let addedCourse: ScheduleCourse;
         if (scheduleIndex === this.schedule.getNumberOfSchedules()) {
-            color = this.schedule.addCourseToAllSchedules(newCourse);
+            addedCourse = this.schedule.addCourseToAllSchedules(newCourse);
         } else {
-            color = this.schedule.addCourse(newCourse);
+            addedCourse = this.schedule.addCourse(newCourse);
```

Comment:
```ts
const addedCourse = 
scheduleIndex === this.schedule.getNumberOfSchedules() ? this.schedule.addCourseToAllSchedules(newCourse) : 
this.schedule.addCourse(newCourse)
```

Optional: doing this allows TypeScript to infer the return type implicitly; downside is that it's harder to read